crocket/src/main/java/com/crocket/model/EntityDrawableVisitor.java

```diff
@@ -37,12 +38,19 @@ private static EntityType getEntityTypeFromClass(Class<?> type) {
     }
 
     private void addDrawableEntity(Entity entity, int rotation) {
+        boolean active = false;
         int xPosition = (int) entity.getxPosition();
         int yPosition = (int) entity.getyPosition();
         int width = entity.getWidth();
         int height = entity.getHeight();
+        IModel model = Model.getInstance();
+        Player activePlayer = model.getActivePlayer();
+        Entity entity2 = activePlayer.getActiveTarget();
```

Comment:
I would suggest renaming this to something more descriptive. Maybe `activeTarget`